releng/setup-env.sh

```diff
@@ -466,7 +466,7 @@ case $host_platform in
     OBJCOPY="${android_toolroot}/bin/${host_toolprefix}objcopy"
     OBJDUMP="${android_toolroot}/bin/${host_toolprefix}objdump"
 
-    CFLAGS="$host_arch_flags -DANDROID -fPIE -ffunction-sections -fdata-sections"
+    CFLAGS="$host_arch_flags -DANDROID -fPIC -ffunction-sections -fdata-sections"
```

Comment:
This already gets added right after the `switch` on line 630, so we can omit this part.

Reply:
Thanks @oleavr . Ah yes, I see that now.  I'll make the change and retest.